### PR TITLE
feat(forgot-password-recovery-page): Add forgot password recovery page(#502)

### LINKED ISSUE #502: [Frontend] Template for Forgot Password recovery
Deliverable:

- [ ] We need a page containing a form with following fields
1. New Password
2. Confirm New Password

Add a validation check for the passwords i.e both the passwords should match, else show the proper error message via toast.
This page should appear on the route `/forgotPassword/{id}`

### PR DESCRIPTION
**Pull request by ankitaugale23**
shows a template for Forgot Password Recovery Page

fixes #502

## Issue that this pull request solves

 Closes: # (502)

## Proposed changes
Template for Forgot Password Recovery Page
### Brief description of what is fixed or changed
Add template of Forgot Password Recovery Page at the route "/forgot-password/{id}"
## Types of changes
Frontend
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots
[https://drive.google.com/file/d/1n6M0tBlgsfExZT_i0JXgdri7BZbBJB3A/view?usp=sharing](url)
**Attaching screen recording**
Please attach the screenshots of the changes made in case of a change in the user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
     "react-simple-card": "^2.0.2",
+    "react-toastify": "^7.0.3",
```
Comment (Kajol-Kumari) [inline frontend/package.json:29]: please don't install as new package for toast as we already have a toast component under `util` folder, use that.

File: frontend/src/pages/ForgotPasswordRecovery/ForgotPasswordRecovery.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  }
+
+  return (
+    <>
+      <div className={style["forgot-password-recovery-section"]}>
+        <div
+          className={`${style["forgot-password-recovery-image"]} ${style["child1"]}`}
+        >
+          <img
+            src="./images/forgot-password-recovery.jpg"
```
Comment (Kajol-Kumari) [inline frontend/src/pages/ForgotPasswordRecovery/ForgotPasswordRecovery.jsx:38]: I can't see this image in this PR. Please upload it under `public/images`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
After user enters same password twice in the form and clicks on submit button, please do the following -
1. Show the toast saying `Password Changed Successfully!`
2. Show a card to the user similar to below :arrow_down: 
Screenshot from 2021-03-16 11-11-23

Just change this card content accordingly here

**Comment by ankitaugale23** [comment]
ok sure. I will do that @Kajol-Kumari

**Comment by ankitaugale23** [comment]
Sure. I will make a PR today only @Kajol-Kumari  with all the requested changes.

File: frontend/src/app.js [hunk 3]
```diff
@@ -122,8 +123,13 @@ const App = () => {
                 <Route
                   exact={true}
                   path="/forgot-password"
-                  component={ForgotPassword}
```
Comment (Kajol-Kumari) [inline frontend/src/app.js:125]: please don't change it it's meant to be `ForgotPassword` Component on;y

File: frontend/src/app.js [hunk 4]
```diff
@@ -122,8 +123,13 @@ const App = () => {
...
+                  component={ForgotPasswordRecovery}
                 />
+                {/* <Route
```
Comment (Kajol-Kumari) [inline frontend/src/app.js:128]: Please uncomment these line.

File: frontend/src/pages/ForgotPasswordRecovery/ForgotPasswordRecovery.jsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+                        onChange={(e) => {
+                          setConfirmPassword(e.target.value);
+                        }}
+                      ></input>
+                    </div>
+
+                    <div className={style["submit-btn"]}>
+                      <Button2
+                        type="submit"
+                        label="Submit"
```
Comment (Kajol-Kumari) [inline frontend/src/pages/ForgotPasswordRecovery/ForgotPasswordRecovery.jsx:114]: Please label it as `Login` as we will be directing the user to Login Form after they click on it

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @ankitaugale23 Thanks for incorporating the suggested changes. Few minor changes required, rest looks good to go :)

File: frontend/src/pages/ForgotPasswordRecovery/ForgotPasswordRecovery.jsx (hunk 5 above)
Comment (ankitaugale23) [reply frontend/src/pages/ForgotPasswordRecovery/ForgotPasswordRecovery.jsx]: I have completed all the changes and for the "forgot password" route I was checking my code on that route and I forgot to update it. Sorry about that. 
Now I have updated all of the things and removed the conflicts too. 
Let me know if there is an issue. Also, the image is there in this"public/images" folder. I am able to see it. Kindly tell me if you are able to see it or not?

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for the quick fix @ankitaugale23 
LGTM :+1:

**Pull request merged**